code/utils.q

```diff
@@ -1,25 +1,28 @@
-
-\d .automl
-
+// code/utils.q - General utility functions
+// Copyright (c) 2021 Kx Systems Inc
+//
 // The purpose of this file is to house utilities that are useful across more
 // than one node or as part of the AutoML fit functionality and graph.
 
-// @kind function
+\d .automl
+
+// @kind data
 // @category utility
-// @fileoverview List of models to exclude
+// @desc List of models to exclude
+// @type symbol[]
 utils.excludeList:`GaussianNB`LinearRegression
 
 // @kind function
 // @category utility
-// @fileoverview Defaulted fitting and prediction functions for AutoML cross
+// @desc Defaulted fitting and prediction functions for AutoML cross
 //   validation and hyperparameter search. Both models fit on a training set
 //   and return the predicted scores based on supplied scoring function.
 // @param func {<} Scoring function that takes parameters and data as input, 
 //   returns appropriate score
-// @param hyperParam {dict} Hyperparameters to be searched
+// @param hyperParam {dictionary} Hyperparameters to be searched
 // @param data {float[]} Data split into training and testing sets of format
 //   ((xtrn;ytrn);(xval;yval))
-// @return {(bool[];float[])} Predicted and true validation values
+// @return {list} Predicted and true validation values
```

Comment:
This should be `(bool[]|float[])` because it's not a mixed list. Returns a list of booleans or floats